src/test/ui/rfc-2091-track-caller/track-caller-attribute.rs

```diff
@@ -0,0 +1,40 @@
+// run-pass
+
+#![feature(const_fn, track_caller)]
```

Comment:
The `const_fn` feature looks unnecessary here?

Reply:
Thanks! Addressed in https://github.com/rust-lang/rust/pull/67137/commits/753fa0ef46eb49042392b1eaa94376802e884bf1